examples/detection/detection.py

```diff
@@ -142,11 +177,21 @@ async def detect(self, frame: rtc.VideoFrame):
         finally:
             self.detecting = False
 
-    async def publish_video(self):
-        track = rtc.LocalVideoTrack.create_video_track("agent-video", self.video_out)
-        options = rtc.TrackPublishOptions()
-        options.source = rtc.TrackSource.SOURCE_CAMERA
-        await self.ctx.room.local_participant.publish_track(track, options)
+    async def send_audio_stream(
+        self, tts_stream: AsyncIterable[agents.tts.SynthesisEvent]
+    ):
+        async for e in tts_stream:
+            if e.type == agents.tts.SynthesisEventType.STARTED:
```

Comment:
Conditions aren't necessary here, but the streams of TTS still need discussion anyway